src/client/Project.ts

```diff
@@ -69,27 +85,41 @@ export default class Project {
     this.isArchived = props.isArchived;
     this.isRecycled = props.isRecycled;
     this.statuses = props.statuses;
+    this.statusIds = props.statusIds;
     this.#client = client;
 
   }
 
+  async createLabel(props: Omit<InitialLabelProperties, "projects">): Promise<Label> {
+
+    return await this.#client.createLabel({...props, projects: [this.id]});
```

Comment:
Create a separate issue to remove "projects" from Label. Needs to be like `createStatus` below